_pages/tutorials/webinars.md

```diff
@@ -22,11 +22,15 @@ Whether you're looking to supercharge your application with readily-usable Machi
 
 Algorithmia's AI Layer and Marketplace make it easy to get Machine Learning up and running in your app. Our Webinar will show you how.
 
-<a href="https://register.gotowebinar.com/rt/6993672803865799683" class="btn btn-default btn-primary">Register Now</a>
+<a href="https://register.gotowebinar.com/rt/6993672803865799683">
```

Comment:
Let's add `target="_blank"` + `rel="noreferrer noopener"` to both of these links while we're at it.